tests/integration/control/pod/test_configure_pod_index.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+import time
+
+class TestConfigurePodIndex():
+    def test_configure_pod_index(self, client, ready_index):
+        time.sleep(10) # Wait a little more, just in case.
+        client.configure_index(ready_index, replicas=1, pod_type='p1.x1')
```

Comment:
Do we need some asserts here?

Reply:
Maybe someday. To start with, I just wanted to see if the request can be made without errors and this was enough to catch the bug we saw ahead of the release last week. Asserting what the server actually does as a result can come later, or possibly be covered in tests owned by platform.